### PR TITLE
Matching gateway timezone and PayPal Issues

### PR DESCRIPTION
**Pull request by tannerm**
#298 #242 #390 #229

### CODE REVIEW

File: includes/gateways/class-rcp-payment-gateway-paypal-express.php [hunk 1]
```diff
@@ -177,7 +177,7 @@ public function process_confirmation() {
 					'VERSION'             => '124',
 					'TOKEN'               => $_POST['token'],
 					'METHOD'              => 'CreateRecurringPaymentsProfile',
-					'PROFILESTARTDATE'    => date( 'Y-m-d\Tg:i:s', strtotime( '+' . $details['subscription']['duration'] . ' ' . $details['subscription']['duration_unit'], time() ) ),
```
Comment (pippinsplugins) [inline includes/gateways/class-rcp-payment-gateway-paypal-express.php]: PayPal is extremely picky with the date format. Test creating a new profile after this change?

Comment (tannerm) [inline includes/gateways/class-rcp-payment-gateway-paypal-express.php]: Yup, I'll be doing testing with PayPal today. It should be the same format, g is just 12 hour (and was causing issues) while H is the 24 hour equivalent.

Comment (pippinsplugins) [inline includes/gateways/class-rcp-payment-gateway-paypal-express.php]: 👍 ping me here once tests are done

Comment (tannerm) [inline includes/gateways/class-rcp-payment-gateway-paypal-express.php]: @pippinsplugins ok, I just tested this with PayPal and it looks good.

**Pull request merged**